### PR TITLE
feat: add GoFr swag

### LINKED ISSUE #1261: GoFr Swag opportunity
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Gofr (https://gofr.dev/)

Q. Where have you seen the announcement (URL)?  
A. : Github repo readme (https://github.com/gofr-dev/gofr)

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing, stickers

Q. Have you received swag from this?  
A. : Yes

Q. How long did it take?  
A. : n/a

Q. Does it ship worldwide?  
A. : Yes

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by Tejas-Sharma-dev**
fix #1261: added the new swag in data.json file.

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

updated data.json for the GoFr swag, please review the changes and let me know if i have missed anything.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -449,6 +449,15 @@
     "dateAdded": "2019-10-10T13:00:00.000Z",
     "tags": ["hacktoberfest", "clothing", "expired"]
   },
+  {
+    "name": "GoFr",
+    "difficulty": "medium",
+    "description": "Contribute to GoFr by submitting a notable pull request or by writing an article. contributions in someway to development or spreading the word about GoFr. receive GoFr T-Shirt and Stickers as a token of appreciation.",
+    "reference": "https://github.com/gofr-dev/gofr",
+    "image": "https://gofr.dev/_next/static/media/complete-gorg-logo.bc9195c4.svg",
```
Comment (aslafy-z) [inline data.json:457]: Please use some other link, these next assets are likely to change path at every build.

File: data.json [hunk 2]
```diff
@@ -449,6 +449,15 @@
     "dateAdded": "2019-10-10T13:00:00.000Z",
     "tags": ["hacktoberfest", "clothing", "expired"]
   },
+  {
+    "name": "GoFr",
+    "difficulty": "medium",
+    "description": "Contribute to GoFr by submitting a notable pull request or by writing an article. contributions in someway to development or spreading the word about GoFr. receive GoFr T-Shirt and Stickers as a token of appreciation.",
```
Comment (aslafy-z) [inline data.json:455]:
```suggestion
    "description": "Contribute to GoFr by submitting a notable pull request or by writing an article. Contributions in someway to development or spreading the word about 'GoFr'. Receive GoFr T-Shirt and Stickers as a token of appreciation.",
```

**Review by aslafy-z — CHANGES_REQUESTED** [review]

File: data.json (hunk 2 above)
Comment (Tejas-Sharma-dev) [reply data.json:455]: why is the build failing?

Comment (Tejas-Sharma-dev) [reply data.json:455]: I'll update the description as you suggested and change the image url to (https://avatars.githubusercontent.com/u/146108433?s=200&v=4). hope it works.

File: data.json (hunk 1 above)
Comment (Tejas-Sharma-dev) [reply data.json:457]: for image URL?

**Comment by Tejas-Sharma-dev** [comment]
I think, somewhere i have made mistake in whitespace. I'm unable to debug. shall i re-submit the PR

**Comment by Tejas-Sharma-dev** [comment]
whitescapes are great trouble maker.

File: data.json [hunk 3]
```diff
@@ -449,6 +449,15 @@
     "dateAdded": "2019-10-10T13:00:00.000Z",
     "tags": ["hacktoberfest", "clothing", "expired"]
   },
+  {
+    "name": "GoFr",
+    "difficulty": "medium",
+    "description": "Contribute to GoFr by submitting a notable pull request or by writing an article. Contributions in someway to development or spreading the word about 'GoFr'. Receive GoFr T-Shirt and Stickers as a token of appreciation.",
+    "reference": "https://github.com/gofr-dev/gofr",
```
Comment (aslafy-z) [inline data.json:456]:
```suggestion
    "reference": "https://github.com/gofr-dev/gofr#-contribute",
```

**Pull request merged**